Makefile

```diff
@@ -25,7 +25,11 @@ ifeq ($(NOSTRETCH), 0)
 	make -f Makefile.work stretch
 endif
 
-clean reset init configure showtag sonic-slave-build sonic-slave-bash :
+clean reset init:
+	@echo "+++ Making $@ +++"
+	make -f Makefile.work $@
+
+configure showtag sonic-slave-build sonic-slave-bash :
```

Comment:
I think 'configure' also only needs to be run once. Can we also add it to the above?

Reply:
configure references SLAVE_BASE, which is distribution dependent. So I left it out.

Reply:
I don't see the need for 'configure' to need the slaves built as a dependency. We can look at this for a future PR.